tools/property-extractor/transformers.py

```
@@ -1606,14 +1606,19 @@ def _resolve_numeric_constant(self, identifier):
             return None
 
         try:
             result = self._constant_resolver.resolve_numeric_constant(identifier)
             if result:
-                return result.get('friendly')
+                friendly = result.get('friendly')
+                logger.info(f"Resolved constant {identifier} = {friendly}")
+                return friendly
+            else:
+                logger.warning(f"Failed to resolve numeric constant: {identifier}")
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)